Test/Utils/CommonTestCode.ps1

```diff
@@ -1,3 +1,4 @@
+. $PSScriptRoot\..\..\CIScripts\Common\Init.ps1
```

Comment:
Changes like this involve explicitely importing files that we use, instead of relying on order of loading them in runtime

Reply:
I'm not sure I get this comment.

Previously, we imported `Init.ps1` only in the top-level "script" PowerShell files. This file is a "library" PowerShell file. Do we want to do it that way?